### PR TITLE
[TFF-170] Use consusd on port 8081 for integration tests

### PR DESCRIPTION
**Pull request by trevinhofmann**
### Summary

This PR uses `consusd` to run the server in the background for integration tests. It uses port 8081 so that we can run the functional tests with port 8080 acting as a proxy to port 8081, resulting in our integration tests.

Sister PRs (merge **this** PR last!):
* TheFourFifths/consus/pull/69 (merge first!)
* TheFourFifths/consus-client/pull/90 (merge PR second!)

### Checklist

- [x] Have you added unit and functional tests as appropriate?
- [x] Did you update/add documentation for new methods or changed functionality?
- [x] Have you merged the target branch down into this one?

### How to Review

1. Merge the sister PRs first.
2. View the integration tests on Travis.
3. Verify that the tests execute and pass.

### Links

- [TFF-170](https://msoese.atlassian.net/browse/TFF-170)
- TheFourFifths/consus/pull/69
- TheFourFifths/consus-client/pull/90

### CODE REVIEW

**Comment by gearedmonkey** [comment]
@thofmann the tests are not passing on travis

**Comment by trevinhofmann** [comment]
Tests should pass once the sister PRs are merged on consus and consus-client.

**Review by mattmahn — COMMENTED** [review]
LGTM assuming the tests pass

**Comment by trevinhofmann** [comment]
Tests fixed. Needed to have Travis install dependencies before installing `consus` globally.

**Review by mattmahn — APPROVED** [review]
#approve

**Review by HPoe2013 — APPROVED** [review]
LGTM

**Review by gearedmonkey — COMMENTED** [review]
LGTM

**Pull request merged**
